automatic/mp3tag/README.md

```diff
@@ -3,3 +3,9 @@
 
 Mp3tag is a powerful and yet easy-to-use tool to edit metadata of common audio formats where it supports ID3v1, ID3v2.3, ID3v2.4, iTunes MP4, WMA, Vorbis Comments and APE Tags.
 
+## Parameters
+- `/NoDesktopShortcut` - Do not create desktop shortcut for Mp3tag
+- `/NoContextMenu` - Do not add Mp3tag to context menu
+
+These parameters can be passed to the installer with the use of `--params`.
+For example: `--params '"/NoDesktopShortcut /NoContextMenu"'`
```

Comment:
Please use a full example, ie include the call to choco, and use the full name of the argument

```suggestion
For example: `choco install --package-parameters='"/NoDesktopShortcut /NoContextMenu"'`
```